cli/storage_inmemory.go

```diff
@@ -0,0 +1,28 @@
+package cli
```

Comment:
this will bring in `repotesting` dependency into the kopia binary which is undesired. Can you move this to a test-only package location?

Reply:
other than this, LGTM

Reply:
good point, yes

Reply:
done in 232d764825432a1b381c78dcbc0922b594dc4a89